doc/source/general/compiling.rst

```diff
@@ -18,7 +18,33 @@ When you try to run your script, the first thing that kOS does is transform your
 
 This process of transforming your script into Machine Language, or "ML" is called "Compiling".
 
-The ":ref:`RUN`" and ":ref:`RUNPATH`" commands do this silently, without telling you. This is why you may have noticed the universe slightly stutter for a moment when you first run your program. Compiling is hard work for the universe to do.
+The :key:`RUN` and :func:`RUNPATH` commands do this silently, without telling
+you.  The :key:`COMPILE` command explicitly compiles the file and saves it for
+future use.
+
+.. note::
+    .. versionchanged:: 1.1.0
+        kOS now performs compilation on a parallel thread (thought to run in a
+        parallel universe).  This means that the universe no longer needs to
+        freeze in order to perform the compilation.  As with the execution of
+        other triggers, compiling from mainline code will not interupt or pause
+        triggers but compiling within a trigger will result in both the mainline
+        and trigger code pausing until the compilation is complete.
```

Comment:
> Given the tone of this document page, I didn't want to break the 4th wall by talking about the underlying system.

Mentioning the threading is *already* breaking the forth-wall.  In the gameworld side of that forth wall, it doesn't look threaded.  My edit was mostly under the assumption you *did* want to break the 4th wall here (because you mentioned the threading that's invisible to game side of the wall), and if you're doing to do that then we should be clear that the threading you're talking about isn't a kOS in-gameworld thing.

I guess my problem is this:
```
print "before".
compile "foo.ks" to foo.ksm".
print "after".
```
Saying "performs compilation on a parallel thread" would give the impression that the above example will print ``after`` right away, BEFORE the compile command is done.  Because we claimed the compile happens in parallel.  I wanted it to be clear that in the not-breaking-4th-wall way of looking at things, compiles still happen in serial just like before.